doc/source/whatsnew/v0.18.0.txt

```diff
@@ -251,6 +251,38 @@ the ``extractall`` method returns all matches.
 
 .. _whatsnew_0180.enhancements.rounding:
 
+
+Changes to str.cat
+^^^^^^^^^^^^^^^^^^
+
+The :ref:`.str.cat <text.cat>` concatenates the members of a Series. Before, if NaN values
+were present in the Series, calling `cat()` on it would return NaN, unlike the rest of the
+`Series.str.*` API. This behavior has been amended to ignore NaN values by default.
+(:issue:`11435`).
+
+A new, friendlier ValueError was also added to protect against the mistake of supplying the
+`sep` as an arg, rather than a kwarg.
+(:issue:`11334`).
+
+.. code-block:: python
```

Comment:
make the first 2 an ipython block so they run, the 3rd can be a code-block (just so you can show the error and not the entire traceback)